docs/source/clustering/calinski_harabasz_score.rst

```diff
@@ -0,0 +1,21 @@
+.. customcarditem::
+   :header: Calinski Harabasz Score
+   :image: https://pl-flash-data.s3.amazonaws.com/assets/thumbnails/clustering.svg
```

Comment:
Does this image exists?

Reply:
Just checked no. Do we have a list of thumbnails available?
Alternatively do we have someone internally that could help create some new thumbnails?

Reply:
I see there... so maybe it is worse to check what we try to use and ask for new...
```
audio_classification.jpg
audio_classification.svg
default.svg
extractive_question_answering.svg
graph_classification.svg
image_classification_multi_label.jpg
image_classification.jpg
image_classification.svg
image_embedder.svg
instance_segmentation.svg
keypoint_detection.svg
multi_label_image_classification.svg
multi_label_text_classification.svg
object_detection.svg
point_cloud_object_detection.svg
point_cloud_segmentation.svg
semantic_segmentation.svg
speech_recognition.svg
style_transfer.svg
summarization.svg
tabular_classification.svg
text_classification.svg
translation.svg
video_classification.svg